app/src/hooks/api/useSecurityApi.ts

```diff
@@ -0,0 +1,42 @@
+import { AxiosInstance } from 'axios';
+import { ISecurityReason } from 'features/datasets/components/security/SecurityReasonCategory';
+import { IArtifact } from 'interfaces/useDatasetApi.interface';
+import { IListPersecutionHarmResponse } from 'interfaces/useSecurityApi.interface';
+
+/**
+ * This hook is used to fetch data from the security api.
+ *
+ * @param {AxiosInstance} axios
+ * @return {*} object whose properties are supported api methods.
+ */
+const useSecurityApi = (axios: AxiosInstance) => {
+  /**
+   * Fetches a list of persecution and harm rules.
+   *
+   * @return {*}  {Promise<IListPersecutionHarmResponse>}
+   */
+  const listPersecutionHarmRules = async (): Promise<IListPersecutionHarmResponse> => {
+    const { data } = await axios.get('/api/security/persecution-harm/list');
+
+    return data;
+  };
+
+  const applySecurityReasonsToArtifacts = async (
```

Comment:
Since the server only cares about the artifact ID and security reason ID, we could save some work and just pass these along instead.